cmd/node-joiner/main.go

```diff
@@ -34,7 +34,22 @@ func main() {
 		Use:   "monitor-add-nodes",
 		Short: "Monitors the configured nodes while they are joining an existing cluster",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return nodejoiner.NewMonitorAddNodesCommand("")
+			wd, err := os.Getwd()
```

Comment:
Should we add a ref about how to build this binary? Like here https://github.com/openshift/installer/blob/master/README.md?plain=1#L26

Reply:
Users aren't going to be using the node-joiner binary directly. They will be using `oc adm add-nodes-image` or `oc adm monitor-add-nodes` as their interface. I don't know if we would want to highlight how to build node-joiner as it may lead to confusion. @andfasano what's your opinion?

Reply:
Agree, the node-joiner it's effectively an intermediate internal binary not meant to be exposed directly.